src/Http/Http.Features/src/IHttpResponseTrailersFeature.cs

```diff
@@ -3,8 +3,18 @@
 
 namespace Microsoft.AspNetCore.Http.Features
 {
+    /// <summary>
+    /// Provides access to Trailer response headers.
```

Comment:
The `Trailer` response header is something else, items in this collection are actually called response trailers.
```suggestion
    /// Provides access to response trailers.
```